tmt-app/src/Components/CareerGuide.js

```diff
@@ -1,12 +1,220 @@
+              flexDirection: {
+                xs: "column",
+                sm: "column",
+                md: "row",
+              },
+            }}
+          >
+            <Typography variant="CustomHeading1">Majors/Careers</Typography>
+            <Typography
+              variant="CustomSubHeading"
+              sx={{
+                marginLeft: {
+                  xs: "0",
+                  sm: "0",
+                  md: "5rem",
+                },
+              }}
+            >
+              Explore majors, careers, and jobs
+            </Typography>
+          </FlexContainer>
+          <Container
+            sx={{
+              display: "flex",
+              flexDirection: {
+                xs: "column",
+                sm: "column",
+                md: "row",
+              },
+              alignItems: {
+                sm: "flex-start",
+                md: "center",
+              },
+              justifyContent: "space-between",
+              marginY: "3rem",
+            }}
+          >
+            <ToggleButtonGroup
+              value={selection}
+              exclusive
+              aria-label="major or career"
+              onChange={handleChoice}
+            >
+              <ToggleButton
+                variant="customToggleStyle"
+                value="Majors"
+                disableRipple
+              >
+                <Typography variant="CustomHeading3">Majors</Typography>
+              </ToggleButton>
+              <ToggleButton variant="customToggleStyle" value="Careers">
+                <Typography variant="CustomHeading3">Careers</Typography>
+              </ToggleButton>
+            </ToggleButtonGroup>
+            <TextField
+              id="filled-search"
+              type="search"
+              sx={{
+                "& .MuiInputBase-input": { padding: "6px" },
+                paddingY: "1.5rem",
+              }}
+              InputProps={{
+                startAdornment: (
+                  <InputAdornment position="start">
+                    <SearchIcon />
+                  </InputAdornment>
+                ),
+                style: {
+                  borderRadius: "30px",
+                },
+              }}
+            />
+          </Container>
+          <FlexContainer
+            sx={{
+              justifyContent: "space-between",
+              flexDirection: {
+                xs: "column",
+                sm: "column",
+                md: "row",
+              },
+              alignItems: {
+                xs: "flex-start",
+                sm: "flex-start",
+                md: "row",
+              },
+            }}
+          >
+            <Typography display="inline" variant="CustomHeading3">
+              Major Result
+            </Typography>
+            <Container sx={{ margin: "0", width: "auto" }}>
+              <Typography variant="CustomSubHeading">Sort By:</Typography>
+              <Autocomplete
```

Comment:
Is there a reason we're using an auto complete over a regular menu?

Clicking through the preview, selecting ascending vs descending for alpha sort has a cursor blinking in the field, seemed strange for selecting preset values